api/organisations/models.py

```diff
@@ -478,3 +478,29 @@ class HubspotOrganisation(models.Model):
     hubspot_id = models.CharField(max_length=100)
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
+
+
+class OrganisationAPIBilling(models.Model):
+    """
+    Tracks API billing for when accounts go over their API usage
+    limits. This model is what allows subsequent billing runs
+    to not double bill an organisation for the same use.
+
+    Even though api_overage is charge per thousand API calls, this
+    class tracks the actual rounded count of API calls that are
+    billed for (i.e., 52000 for an account with 52233 api calls).
```

Comment:
I was thinking we should under charge the usage to the nearest thousand. I would be kinda ticked off if I found out one of my providers were nickel and diming me by rounding up!

Reply:
I'd have said it makes most sense to round up since all our plans are 'up to X amount' but, since 1000 api requests will be about ~$0.07 I really don't care. Maybe just add an extra line here to say that it was a conscious decision to round down so that it's not confusing in the future.

Reply:
Ok I've added an extra line to the docstring.